static/graphs/obsidian.js

```diff
@@ -0,0 +1,277 @@
+const MINIMAL_NODE_SIZE = 8
```

Comment:
Very minor thing, but could you run Prettier on this file? I am seeing inconsistency with usage of semicolons for example.

Reply:
I spotted this too. Is there a `.prettierrc` in the repo?

Reply:
I think not. I used to have it in projects where I wanted to change some settings, but if I am not planning to... should it be just empty `.prettierrc` with `{}` as content?

Reply:
@tchayen maybe the defaults for either `1.x` or `2.x` prettier would be a good idea? https://prettier.io/docs/en/configuration.html#basic-configuration this seems a good place to start.

But empty could work.

To me it feels that at least putting _something_ will mean we don't get odd scenarios like when @ianjsikes saved over a file that I'd saved before with a different version of prettier.